neural_compressor/torch/algorithms/base_algorithm.py

```diff
@@ -80,6 +76,30 @@ def convert(self, model: torch.nn.Module, *args: Any, **kwargs: Any):
         """
         raise NotImplementedError("{} doesn't implement `convert` function. ".format(self.__class__.__name__))
 
+    def quantize(self, model: torch.nn.Module, *args: Any, **kwargs: Any):
+        """Quantizes a given float model.
+
+        Args:
+            model (torch.nn.Module): The float model to be quantized.
+
+        Returns:
+            A quantized model.
+        """
+        run_fn = kwargs.get("run_fn", None)
+        run_args = kwargs.get("run_args", None)
+        assert run_fn is not None, (
+            "Can't find run_func. Please provide run_func to quantize API "
+            "or overwrite quantize member function in your Quantizer class."
+        )
+
+        model = self.prepare(model, *args, **kwargs)
+        if run_args:
+            run_fn(model, *run_args)
+        else:
+            run_fn(model)

```

Comment:
How do we handle autoround case? Autoround will perform calibration within AutoRound instead of INC.

Reply:
Discussed offline, for details refer https://github.com/intel/neural-compressor/pull/1763